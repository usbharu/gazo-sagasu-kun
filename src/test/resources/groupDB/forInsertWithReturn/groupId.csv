"id","name"
1,"upright"
2,"whatever"
3,"know"
