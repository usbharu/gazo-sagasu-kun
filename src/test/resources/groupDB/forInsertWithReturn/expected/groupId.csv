"id","name"
1,"upright"
2,"whatever"
3,"know"
4,"insertWithReturn"
